packages/salesforcedx-sobjects-faux-generator/src/describe/sObjectDescribe.ts

```diff
@@ -29,24 +36,59 @@ export class SObjectDescribe {
    * @param type SObjectCategory
    * @returns string[] containing the sobject names
    */
-  public async describeGlobal(type: SObjectCategory): Promise<string[]> {
+  public async describeGlobal(
+    type: SObjectCategory,
```

Comment:
@lcampos nit: It might be a little clearer if this is name "category" instead of "type".

Reply:
I agree that's a better name, I'll change it.